### PR TITLE
PE-256

### PR DESCRIPTION
**Pull request by AuraAlba**
@ryangadams

Remove qa logo form header and add on footer.
footer-bp-befpre
footer-bp-after (1)

Add space on header links
header-bp-before
header-link-bp-after

Remove hover for courses list on index
course-bp-beforecourse-bp-after

Change image for banner.
bp-banner-before
banner-image-ater

### CODE REVIEW

File: edx-platform/bp-theme/lms/templates/course.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<%def name="online_help_token()"><% return "course" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.urls import reverse
+from six import text_type
```
Comment (Squirrel18) [inline edx-platform/bp-theme/lms/templates/course.html:6]: Please, @amalbas Sort the imports according to [PEP8](https://www.datacamp.com/community/tutorials/pep8-tutorial-python-code#import).

**Comment by ryangadams** [comment]
@amalbas 

Some feedback,

1. can you remove the proversity logo and put the QA one in its place?
2. the request to adjust the spacing was to even it up rather than to add more. Could you reduce the gap a little bit, but ensure that the spacing is consistent. it's close!
3. The after shot of the hero image shows the wrong text, I assume that's because it's on your machine rather than on current production. I think the image might need to be darkened a little to ensure the text is readable.

**Comment by AuraAlba** [comment]
@ryangadams I've updated the images according to your feedback. Please confirm everything is good to continue.
Yes, text on the banner is wrong because is my local environment. But in production is ok.

**Comment by ryangadams** [comment]
Looks good. Feel free to deploy.

**Pull request merged**